java/src/jmri/jmrix/loconet/LnReporterManager.java

```diff
@@ -103,16 +103,24 @@ public String getEntryToolTip() {
     @Override
     public void message(LocoNetMessage l) {
         // check message type
-        if (l.getOpCode() != 0xD0) {
-            return;
+        int addr;
+        switch (l.getOpCode()) {
+            case 0xD0:
+                if ((l.getElement(1) & 0xC0) == 0) {
+                    addr = (l.getElement(1) & 0x1F) * 128 + l.getElement(2) + 1;
+                    break;
+                }
+                return;
+            case 0xe5:
+                if (l.getElement(1) == 0x09 && l.getElement(2) == 0x00) {
```

Comment:
Must also decode upper nibble of <PXCT1> and upper nibble of <PXCT2>, else other types of LocoNet message will be decoded as transponding messages.